### PR TITLE
Support TUNING in schedule section.

### PR DESCRIPTION
**Pull request by totto82**

### CODE REVIEW

**Comment by atgeirr** [comment]
jenkins build this please

**Comment by atgeirr** [comment]
Thanks, merging.

File: opm/simulators/timestepping/AdaptiveTimeStepping.hpp [hunk 1]
```diff
@@ -97,6 +97,16 @@ namespace Opm {
 
         void setSuggestedNextStep(const double x) { suggested_next_timestep_ = x; }
 
+        void updateTUNING(const Tuning& tuning, size_t time_step) {
+            restart_factor_ = tuning.getTSFCNV(time_step);
+            growth_factor_ = tuning.getTFDIFF(time_step);
+            max_growth_ = tuning.getTSFMAX(time_step);
+            max_time_step_ = tuning.getTSMAXZ(time_step);
+            suggested_next_timestep_ = tuning.getTSINIT(time_step);
```
Comment (atgeirr) [inline opm/simulators/timestepping/AdaptiveTimeStepping.hpp]: I think this had an unwanted side effect: if there is no change in TUNING, we will still set the next timestep to be the (typically small) timestep from TUNING. So every report step must start with 0.1 day for example. To handle this we must know if TUNING was given *again* for the current step or not.

Comment (totto82) [reply opm/simulators/timestepping/AdaptiveTimeStepping.hpp]: Ok. I can add it as an event in the parser.

**Pull request merged**